pandas/core/base.py

```diff
@@ -652,8 +653,10 @@ def transpose(self, *args, **kwargs):
         nv.validate_transpose(args, kwargs)
         return self
 
-    T = property(transpose, doc="Return the transpose, which is by "
-                                "definition self.")
+    T = property(transpose, doc="""
+                                Return the transpose, which is by
+                                definition self.
+                                """)
```

Comment:
can you use `\n` instead of line breaks please

Reply:
Does this fix it?